src/app/controller/pipes/vessel-tooltip.pipe.ts

```diff
@@ -8,15 +7,20 @@ import { Vessel } from "../../model/portCall/vessel";
 })
 export class VesselTooltipPipe implements PipeTransform {
 
-  transform(vessel: Vessel): string {
+  transform(vessel: Vessel, milesRemainingToDestination: number): string {
```

Comment:
I have called it `milesToDestinationPort` in the API specs

Reply:
This is in extension of discussion I brought up yesterday about alignment of fields naming. `milesRemainingToDestination `is the given field name in the backend.